<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWidget</class>
 <widget class="QDockWidget" name="PlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>159</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>No music selected</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_Open">
        <property name="toolTip">
         <string>Select a music</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/select-music.png</normaloff>:/icons/select-music.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_PlayPause">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-playback-start">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slider_Duration">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Duration">
        <property name="text">
         <string notr="true">--:--/--:--</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="QLineEdit" name="le_Artist"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_Title"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Artist:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Album:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_Album"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Editor:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="le_Editor"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
